packages/foundry-react-ui/src/components/InteractionFeedback/InteractionFeedback.tsx

```diff
@@ -36,7 +36,8 @@ export type InteractionFeedbackProps = {
 };
 
 const defaultInterpolationFunctions = {
-  r: (r: any) => r.to((val: string) => `${Math.abs(parseFloat(val))}`),
+  r: (r: any) => r.to((val: string) => `${Math.abs(parseFloat(val).toFixed(2))}`),
+  opacity: (opacity: any) => opacity.to((val: string) => `${parseFloat(val).toFixed(2)}`),
```

Comment:
Similarly, we probably don't need the opacity string to be wrapped in a template string
 ```suggestion
  opacity: (opacity: any) => opacity.to((val: string) => parseFloat(val).toFixed(2),
```

Reply:
Thanks for the patience. I have not used React since 2015 and even then I think I made an increment counter.